gh-actions/build-changelog/index.js

```diff
@@ -0,0 +1,31 @@
+const core = require('@actions/core');
+const { GitHub } = require('@actions/github');
+
+async function run() {
+  try {
+    const myToken = core.getInput('GITHUB_TOKEN');
+
+    const tag = core.getInput('tag_name', { required: true });
+    const label = tag.substr(10, tag.length - 1);
+
+    const query = `type:pr+label:${label}`;
+
+    const octokit = new GitHub(myToken);
```

Comment:
```suggestion
    const myToken = core.getInput('GITHUB_TOKEN');
    const tag = core.getInput('tag_name', { required: true });
    const label = tag.substr(10, tag.length - 1);
    const query = `type:pr+label:${label}`;
    const octokit = new GitHub(myToken);
```

This is more of a style observation/something I think about sometimes. Since all of these are just one-liner variable declarations, does it make more sense to have extra new lines between some of them, but not all? I think it makes sense to logically group declarations/statements, but sometimes it's just a bunch of variables you need to declare before you do the actual work later in the function body. 

Nothing right or wrong here, just thinking out loud a little bit.